docker/1.3.0/final/Dockerfile.cpu

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:16.04
+ARG framework_installable
+ARG py_version
+ARG framework_support_installable=sagemaker_mxnet_container-2.0.0.tar.gz
+
+# Validate that arguments are specified
+RUN test $framework_installable || exit 1 && \
+    test $py_version || exit 1
+
+WORKDIR /tmp
+
+RUN apt-get update && \
+    apt-get -y install build-essential libopencv-dev libopenblas-dev libjemalloc-dev libgfortran3 \
+    python-dev python3-dev wget curl nginx
```

Comment:
Given that you are not building anything, change {libname}-dev to just {libname}. It saves a lot of image size. Remove build-essential as well.